src/check.js

```diff
@@ -84,19 +151,54 @@ const renovateIsConfigured = {
   link: 'https://github.com/konnectors/docs/blob/master/status.md#renovate'
 }
 
-const repoShouldHave4Branches = {
+const assetsDirIsConfigured = {
   fn: (info, assert) => {
-    for (let branch of ['master', 'build', 'latest', 'prod']) {
-      const fullBranch = info.git.remote + '/' + branch
-      assert(
-        info.git.branches.indexOf(fullBranch) > -1,
-        `Repository should have branch ${branch}`
-      )
-    }
+    assert(
+      fs.existsSync(path.join(info.repository, 'assets')),
+      'The assets directory should exist'
+    )
+    assert(
+      info.webpackConfig.indexOf(`from: 'assets'`) > -1,
+      'The assets directory should be configured in webpack'
+    )
+    const iconPath = path.join(info.repository, 'assets', info.manifest.icon)
+    assert(
+      fs.existsSync(iconPath),
+      `The icon specified in the manifest should exist : ${iconPath}`
+    )
+    return true
   },
-  nickname: '4branches',
-  message: 'Repository should have 4 branches.',
-  link: 'https://github.com/konnectors/docs/blob/master/status.md#git-branch'
+  nickname: 'assets',
+  message: 'The ./assets directory should exist and be configured in webpack'
+}
+
+const manifestAndPackageJsonSameVersion = {
```

Comment:
Would it make sense to just remove the version from the `package.json` file? Or are we supposed to install connectors through npm at some point?

Reply:
No, the registry also checks the version in `package.json` file (I tried :-))